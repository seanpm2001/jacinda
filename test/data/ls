total 216
-rw-r--r--   1 vanessa  staff   4375 Oct 30 17:58 CHANGELOG.md
-rw-r--r--@  1 vanessa  staff  34523 Mar  2  2024 COPYING
-rw-r--r--   1 vanessa  staff    111 Nov 17 07:43 DESIGN.md
-rw-r--r--   1 vanessa  staff   7208 Nov 17 09:30 EXAMPLES.md
-rw-r--r--   1 vanessa  staff    133 Nov  7 11:27 HASKELL-EXAMPLES.md
-rw-r--r--@  1 vanessa  staff    673 Mar  2  2024 LICENSE
-rw-r--r--   1 vanessa  staff   4381 Oct 29 16:18 Makefile
-rw-r--r--   1 vanessa  staff   2620 Oct 30 22:45 README.md
-rw-r--r--   1 vanessa  staff   6564 Oct 30 15:59 ROSETTA.md
-rw-r--r--   1 vanessa  staff   6067 Nov 17 09:14 TODO.md
drwxr-xr-x   3 vanessa  staff     96 Oct 11 12:38 bash
drwxr-xr-x   4 vanessa  staff    128 Nov 17 18:31 bench
-rw-r--r--   1 vanessa  staff    251 Nov 17 09:14 cabal.project
-rw-r--r--   1 vanessa  staff     22 Oct 18 08:32 cabal.project.local
drwxr-xr-x   4 vanessa  staff    128 Oct  5 15:27 csv
drwxr-xr-x   7 vanessa  staff    224 Nov 17 18:47 doc
drwxr-xr-x   4 vanessa  staff    128 Nov 17 09:14 docs
drwxr-xr-x  55 vanessa  staff   1760 Nov 17 14:00 examples
-rw-r--r--   1 vanessa  staff   3909 Nov 17 18:44 jacinda.cabal
drwxr-xr-x  12 vanessa  staff    384 Oct 29 16:18 lib
drwxr-xr-x   4 vanessa  staff    128 Nov 17 13:58 man
drwxr-xr-x   3 vanessa  staff     96 Oct 24 11:43 prelude
drwxr-xr-x  19 vanessa  staff    608 Nov 17 18:40 src
drwxr-xr-x   6 vanessa  staff    192 Nov 17 18:42 test
drwxr-xr-x   7 vanessa  staff    224 Nov 17 09:14 tex
drwxr-xr-x   7 vanessa  staff    224 Nov 17 10:16 vscode
-rw-r--r--   1 vanessa  staff    252 Oct 23 23:55 weeder.toml
drwxr-xr-x   3 vanessa  staff     96 Nov 17 09:31 x
drwxr-xr-x   3 vanessa  staff     96 Nov 11 10:46 zsh
