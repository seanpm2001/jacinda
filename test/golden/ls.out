73.82 kB
